src/odemis/util/test/conversion_test.py

```diff
@@ -314,6 +315,23 @@ def test_get_tile_md_pos(self):
         tile_md_pos = get_tile_md_pos((1, 1), (TILE_SIZE, TILE_SIZE), tile, origda)
         numpy.testing.assert_almost_equal(tile_md_pos, [0.00065225309200, -0.00051098638647])
 
+    def test_json_numpy(self):
+
+        # Lots of types which are not supported by default
+        obj = {"int": numpy.int64(65),  # Works on Python 2 without trick
+               "float": numpy.float64(-64.1),  # Actually works without trick
+               "2d": numpy.array([[1, 2], [3, 4]])
+        }
+
+        s = json.dumps(obj, cls=JsonExtraEncoder)
+
+        # decoding it back is not going to bring back the numpy types, but
+        # something equivalent
+        dec_obj = json.loads(s)
+        self.assertEqual(dec_obj["int"], 65)
+        self.assertEqual(dec_obj["float"], -64.1)
+        self.assertEqual(dec_obj["2d"], [[1, 2], [3, 4]])
+
```

Comment:
It looks as if tests for complex values, sets, and bytes are not implemented in this PR. Maybe this should also be build since the class "JsonExtraEncoder" does add json.dumps functionality for these types.